src/pages/home/page.jsx

```diff
@@ -2,16 +2,10 @@ import React from "react";
 import { getData } from "../../common/request";
 
 
-let HomePage = React.createClass({
-  statics: {
-    fetchData(params) {
-      return getData("/home");
-    }
-  },
-
+export default class HomePage extends React.Component {
   componentWillMount() {
     console.log("[HomePage] will mount with server response: ", this.props.data.home);
```

Comment:
It should: https://github.com/facebook/react/blob/b4962eff43dbf9dc6f85064ea6bbfb444b5557df/src/modern/class/__tests__/ReactES6Class-test.js#L132-L148